command/base.go

```diff
@@ -393,6 +394,13 @@ func (c *BaseCommand) flagSet(bit FlagSetBit) *FlagSets {
 					"This can be specified multiple times.",
 			})
 
+			f.BoolVar(&BoolVar{
+				Name:    "non-interactive",
+				Target:  &c.flagNonInteractive,
+				Default: false,
```

Comment:
What are your thoughts on making this `flagInteractive` instead of `flagNonInteractive`? I find it easier to reason about the positive case, especially since it's the default, rather than having a double negative.

Reply:
I don't like the double negative either. I think we want to have interactive behaviour by default. If we use `flagInteractive`, then the default value would be true. Then, passing `-interactive` to CLI should be interpreted as negative to cover a non-default behaviour which is confusing.

Reply:
Yeah, it's awkward either way, but I agree with Hamid: tools that may be used interactively should default to that, since it's not hard for a script to add another argument to a command, but we don't want to impose that on the user.  

We could add it as default true, then the user would have to say -interactive=false, which wouldn't be too bad.  But I suspect -non-interactive is more intuitive.